js/foundation.util.touch.js

```diff
@@ -140,15 +140,10 @@ class Touch {}
   };
 
   Touch.init = function($) {
```

Comment:
I'd definitely advocate for this. Also, now that the codebase is in a place where the source files can (probably) take any shape, there's more options to bring in external dependencies. For Webpack users, as long as they're declared as Foundation dependencies in `package.json`, they get pulled in automatically. And then those dependencies can possibly be packed in for the bundled version.

I think at its outset, Foundation 6 had a problem with reinventing the wheel in a few cases, and one of the reasons was because we were hesitant to bring in too many non-jQuery dependencies, lest the setup process become too involved.

Hammer would be a good replacement. I'd probably also say the sticky plugin could be replaced with sticky-kit.